packages/config/src/layer2s/zkfair.ts

```diff
@@ -18,20 +21,34 @@ export const zkfair: Layer2 = {
     provider: 'Polygon',
     links: {
       websites: ['https://zkfair.io/'],
-      apps: [],
+      apps: ['https://wallet.zkfair.io/'],
       documentation: ['https://docs.zkfair.io/'],
-      explorers: [],
-      repositories: [],
+      explorers: ['https://scan.zkfair.io/'],
+      repositories: ['https://github.com/ZKFair'],
       socialMedia: ['https://twitter.com/ZK_fair'],
```

Comment:
Seems like this account is suspended, maybe use this: https://twitter.com/ZKFCommunity ?

Reply:
nice catch